src/config/index.ts

```diff
@@ -7,24 +7,24 @@ export const DOCKER_LOCAL_PORT = parseInt(process.env.DOCKER_LOCAL_PORT, 10)
 export const PORT = parseInt(process.env.PORT, 10)
 export const POSTGRES_PORT = parseInt(process.env.POSTGRES_PORT, 10)
 export const JWT_SECRET = process.env.JWT_SECRET ?? 'USER-AUTH'
-export const DB_HOST_TYPE = process.env.DB_HOST_TYPE
-export const DEV_DATABASE_URL = process.env.DEV_DATABASE_URL
-export const TEST_DATABASE_URL = process.env.TEST_DATABASE_URL
-export const PROD_DATABASE_URL = process.env.PROD_DATABASE_URL
-export const GOOGLE_CLIENT_ID = process.env.GOOGLE_CLIENT_ID
-export const GOOGLE_CLIENT_SECRET = process.env.GOOGLE_CLIENT_SECRET
+export const { DB_HOST_TYPE } = process.env
+export const { DEV_DATABASE_URL } = process.env
+export const { TEST_DATABASE_URL } = process.env
+export const { PROD_DATABASE_URL } = process.env
+export const { GOOGLE_CLIENT_ID } = process.env
+export const { GOOGLE_CLIENT_SECRET } = process.env
```

Comment:
```suggestion
export const { DB_HOST_TYPE, DEV_DATABASE_URL, TEST_DATABASE_URL, PROD_DATABASE_URL, GOOGLE_CLIENT_ID, GOOGLE_CLIENT_SECRET} = process.env
```
this can be combined into single export